keyboards/ryloo_studio/m0110/m0110.h

```diff
@@ -58,21 +58,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤
  * │30      │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3d        │
  * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
- * │40  │41  │42  │46                      │49  │4a  │4c  │4d  │
+ * │40  │41  │42  │47                      │49  │4a  │4c  │4d  │
  * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
 */
 #define LAYOUT_60_ansi( \
     k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c,      k0e, \
     k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d,      \
     k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b,      k2d,      \
     k30,      k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b,      k3d,      \
-    k40, k41, k42,                k46,           k49, k4a,           k4c, k4d  \
+    k40, k41, k42,                     k47,      k49, k4a,           k4c, k4d  \
 ) { \
     { k00,  k01,   k02,  k03,   k04,   k05,   k06,  k07,   k08,   k09,  k0a,  k0b,   k0c,   KC_NO, k0e   }, \
     { k10,  k11,   k12,  k13,   k14,   k15,   k16,   k17,  k18,   k19,  k1a,  k1b,   k1c,   k1d,   KC_NO }, \
     { k20,  k21,   k22,  k23,   k24,   k25,   k26,   k27,  k28,   k29,  k2a,  k2b,   KC_NO, k2d,   KC_NO }, \
     { k30,  KC_NO, k32,  k33,   k34,   k35,   k36,   k37,  k38,   k39,  k3a,  k3b,   KC_NO, k3d,   KC_NO }, \
-    { k40,  k41,   k42,  KC_NO, KC_NO, KC_NO, k46,  KC_NO, KC_NO, k49,  k4a,  KC_NO, k4c,   k4d,   KC_NO }  \
+    { k40,  k41,   k42,  KC_NO, KC_NO, KC_NO, KC_NO, k47, KC_NO, k49,  k4a,  KC_NO, k4c,   k4d,   KC_NO }  \
```

Comment:
Ah thank you for catching this.  The space bar should be mapped to k47 instead of k46 like in standard layout.  Sorry about this.  I only tested `LAYOUT_60_ansi` with my tweezer as I use a slightly different layout myself: https://github.com/newtonapple/qmk_firmware/blob/dev/keyboards/ryloo_studio/m0110/keymaps/newtonapple/config.h#L41.  But, even in my own layout (the same bottom row as), I have the space bar mapped to k47.  

```suggestion
    { k40,  k41,   k42,  KC_NO, KC_NO, KC_NO, KC_NO, k47,  KC_NO, k49,  k4a,  KC_NO, k4c,   k4d,   KC_NO }  \
```

Reply:
okay, I added your suggestion, but no idea who can approve and merge this PR here ;D